week-2/G-conditionals-2/exercise-4.js

```diff
@@ -9,7 +9,9 @@
 */
 
 function containsCode(sentence) {
-
+  if (sentence.includes("code")){
+    return true;
+  } return false;
```

Comment:
Missing `else` :) But again it's possible to just return one boolean and get rid of `if/else`! :)